Sources/AppCoinsSDK/Domain/Entities/AnalyticsUserProperties.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  AnalyticsUserProperties.swift
+//  
+//
+//  Created by Graciano Caldeira on 11/07/2024.
+//
+
+import Foundation
+
+public struct AnalyticsUserProperties: Codable {
+    
+    let applicationBundleID: String
```

Comment:
Call this 'package', it is how we are referring to it in the rest of the code base